aldjemy/core.py

```diff
@@ -73,6 +73,17 @@ def get_meta():
     return Cache.meta
 
 
+def get_tables():
```

Comment:
This is new function, but I do not see any use of it.

Reply:
I removed this function in #70, but it is documented in the README. Alternative to adding it back, we could remove the reference fr the README and `__init__.__all__`

Reply:
I removed this function and the documentation of it.